### PR TITLE
[migrations]: add phase and VMI name to the VMIM CLI output

### PR DESCRIPTION
**Pull request by enp0s3**
Signed-off-by: Igor Bezukh <ibezukh@redhat.com>

**What this PR does / why we need it**:
In a large scale environment one may find a lots of VMIM resources. When it comes to CLI it is a tedious
task to manually determine the phase of each such resource and the VMI related to it.

Therefore we expose the VMI and PHASE columns

Fixes #
https://bugzilla.redhat.com/show_bug.cgi?id=2017802

**Special notes for your reviewer**:

**Release note**:
```release-note
Added PHASE and VMI columns for the 'kubectl get vmim' CLI output
```

### CODE REVIEW

File: tests/migration_test.go [hunk 1]
```diff
@@ -2954,6 +2954,60 @@ var _ = Describe("[Serial][rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][leve
 			Expect(imageIDs).To(HaveKeyWithValue(container.Name, digest), "expected image:%s for container %s to be the same like on the source pod but got %s", container.Image, container.Name, imageIDs[container.Name])
 		}
 	})
+
+	Context("CLI output", func() {
```
Comment (xpivarc) [inline tests/migration_test.go:2958]: Wouldn't this be better fit in `kubectl_test.go`?

Comment (enp0s3) [reply tests/migration_test.go:2958]: @xpivarc Hi. Thank you for the notice! I was also thinking to put the test there but wasn't completely sure where it fits most. The best solution is probably to have a dedicated CLI section of e2e, but thats for the follow up.

Comment (enp0s3) [reply tests/migration_test.go]: @xpivarc Done

File: tests/kubectl_test.go [hunk 2]
```diff
@@ -140,4 +144,69 @@ var _ = Describe("[sig-compute]oc/kubectl integration", func() {
 
 	})
 
+	FDescribe("VM instance migration", func() {
+		var virtClient kubecli.KubevirtClient
+
+		runVMIAndExpectLaunch := func(vmi *v1.VirtualMachineInstance, timeout int) *v1.VirtualMachineInstance {
```
Comment (xpivarc) [inline tests/kubectl_test.go:150]: Not sure if this is worth as we use it only once.

File: tests/kubectl_test.go [hunk 3]
```diff
@@ -140,4 +144,69 @@ var _ = Describe("[sig-compute]oc/kubectl integration", func() {
...
+				Eventually(func() error {
+					migrationCreated, err = virtClient.VirtualMachineInstanceMigration(migration.Namespace).Create(migration)
+					return err
+				}, tests.MigrationWaitTime, 1*time.Second).Should(Succeed(), "migration creation should succeed")
+				migration = migrationCreated
+
+				tests.ExpectMigrationSuccess(virtClient, migration, tests.MigrationWaitTime)
+
+				k8sClient := tests.GetK8sCmdClient()
+				result, _, err := tests.RunCommand(k8sClient, "get", "vmim", migration.Name)
```
Comment (xpivarc) [inline tests/kubectl_test.go:184]: Maybe we should write a wrapper around this. (not necessary in this PR of course :) )

File: tests/kubectl_test.go [hunk 4]
```diff
@@ -140,4 +144,69 @@ var _ = Describe("[sig-compute]oc/kubectl integration", func() {
 
 	})
 
+	FDescribe("VM instance migration", func() {
```
Comment (xpivarc) [inline tests/kubectl_test.go:147]: Left over.

**Review by xpivarc — CHANGES_REQUESTED** [review]
Looks good.

File: tests/kubectl_test.go (hunk 3 above)
Comment (enp0s3) [reply tests/kubectl_test.go:184]: @xpivarc Sure. Do you mean to wrap the `RunCommand` retires or to wrap also the `ExpectMigrationSuccess`?

Comment (xpivarc) [reply tests/kubectl_test.go:180]: Just the retry. It's all over the place now and it would be safer to have it contained.

Comment (enp0s3) [reply tests/kubectl_test.go:180]: @xpivarc Sure. Got you. Lets do it in the follow up PR

**Review by dhiller — COMMENTED** [review]
/approve

**Pull request merged**